sdk/healthinsights/Azure.Health.Insights.CancerProfiling/api/Azure.Health.Insights.CancerProfiling.netstandard2.0.cs

```diff
@@ -6,10 +6,10 @@ public partial class CancerProfilingClient
         public CancerProfilingClient(System.Uri endpoint, Azure.AzureKeyCredential credential) { }
         public CancerProfilingClient(System.Uri endpoint, Azure.AzureKeyCredential credential, Azure.Health.Insights.CancerProfiling.CancerProfilingClientOptions options) { }
         public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
-        public virtual Azure.Operation<System.BinaryData> InferCancerProfile(Azure.WaitUntil waitUntil, Azure.Core.RequestContent content, string repeatabilityRequestId = null, System.DateTimeOffset? repeatabilityFirstSent = default(System.DateTimeOffset?), Azure.RequestContext context = null) { throw null; }
-        public virtual Azure.Operation<Azure.Health.Insights.CancerProfiling.OncoPhenotypeResult> InferCancerProfile(Azure.WaitUntil waitUntil, Azure.Health.Insights.CancerProfiling.OncoPhenotypeData oncoPhenotypeData, string repeatabilityRequestId = null, System.DateTimeOffset? repeatabilityFirstSent = default(System.DateTimeOffset?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Operation<System.BinaryData>> InferCancerProfileAsync(Azure.WaitUntil waitUntil, Azure.Core.RequestContent content, string repeatabilityRequestId = null, System.DateTimeOffset? repeatabilityFirstSent = default(System.DateTimeOffset?), Azure.RequestContext context = null) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Operation<Azure.Health.Insights.CancerProfiling.OncoPhenotypeResult>> InferCancerProfileAsync(Azure.WaitUntil waitUntil, Azure.Health.Insights.CancerProfiling.OncoPhenotypeData oncoPhenotypeData, string repeatabilityRequestId = null, System.DateTimeOffset? repeatabilityFirstSent = default(System.DateTimeOffset?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Operation<System.BinaryData> InferCancerProfile(Azure.WaitUntil waitUntil, Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
```

Comment:
I think this is the right fix but this would cause breaking change to health insight.  @m-nash, do you have any concern for this change?

Reply:
> I think this is the right fix but this would cause breaking change to health insight. @m-nash, do you have any concern for this change?

They are all beta versions. I think it's OK to change it.

Reply:
yeah since they are in beta this is fine and expected

Reply:
Hi,
It is ok by us that the repeatability headers will be set internally.
Related to the above change, I see that the Sync method was generated but the Async method dropped.
Any reason why?

Also, a question: is there an option to set the repeatability headers as context policy? (not see it used, it is just my curiosity)

Reply:
Hi @asaflevi-ms are you referring to [ClinicalMatchingClient](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/healthinsights/Azure.Health.Insights.ClinicalMatching/src/Generated/ClinicalMatchingClient.cs)? Which method do you refer to?
There are 4 methods being changed. 2 of them are sync methods, and other 2 are async methods. 

As for the policy, the answer is yes. Users are free to set a pipeline policy through `RequestContext`. See here for example: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/RequestContext.md#add-policy-for-request

Reply:
Hi @archerzz 
My mistake, I missed it.
I reviewed HealthInsights clients (ClinicalMatchingClient and CancerProfilingClient).
Approved by me.